src/http/controllers/dfract.controller.ts

```diff
@@ -5,6 +5,7 @@ import { AssetService } from '@app/services';
 import { DataResponse, DataResponseMetadata, AssetInfo, AssetHistorical } from '@app/http/responses';
 import { ExplorerRequest } from '@app/utils';
 import { AssetRequest } from '@app/http/requests';
+import { DefaultTake } from '../decorators';
```

Comment:
My eyes... :D

Reply:
How did you catch that, I fixed it already :D in this commit https://github.com/lum-network/chain-bridge/pull/100/commits/7388f13007302c68de3575370f6868b09d6ff730